CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 # go-ipfs changelog
 
-## 0.4.12-rc2 2017-10-26
+## 0.4.12 2017-11-09
```

Comment:
Do we want to mention badger breaking changes since the last release?

Reply:
Oh yeah, we should note that. @magik6k people should be able to use your tool soon, right?

Reply:
It doesnt have to block the release since its an experimental thing, but we can let people know such a tool is coming soon.

Reply:
Yep, I'll we just merged https://github.com/ipfs/badgerds-upgrade/pull/1, It works reasonably well for me(i.e. I didn't manage to break it) We can point people there from changelog or make a descriptive error in ds-badger if you don't mind waiting till tomorrow.

Reply:
> We can point people there from changelog or make a descriptive error in ds-badger if you don't mind waiting till tomorrow.

Also update docs/experimental-features.md with that info

Reply:
Done in https://github.com/ipfs/go-ipfs/pull/4379